### PR TITLE
Code styles fixes in feedback implementation

### PR DESCRIPTION
**Pull request by Alroniks**
I found out it when I worked on [BSA-104](https://oat-sa.atlassian.net/browse/BSA-104).

"Use" statements in these files includes classes that already defined in that namespace, it may trigger unexpected errors in the future. Also, I fixed some other issues, mostly related to code style and common sense.

### CODE REVIEW

File: model/qti/feedback/ModalFeedback.php [hunk 1]
```diff
@@ -47,18 +41,14 @@ class ModalFeedback extends Feedback
...
         );
     }
     
     public function toForm()
     {
-        $formContainer = new ModalFeedback($this);
-        return $formContainer->getForm();
+        return (new self($this))->getForm();
     }
 }
```
Comment (shpran) [inline model/qti/feedback/ModalFeedback.php:54]: Are we using this method?
And I think that this one won't work, its parents `__construct` expects an array, we will get the error here: https://github.com/oat-sa/extension-tao-itemqti/blob/ee1d800d983e1cb2bac8fc0e7dbb5b2465c36877/model/qti/Element.php#L139

File: model/qti/feedback/Feedback.php [hunk 2]
```diff
@@ -90,7 +95,7 @@ public function isIdentifierAvailable($newIdentifier)
                 $collection = $relatedItem->getIdentifiedElements();
 
                 try {
-                    $feedback = $collection->getUnique($newIdentifier, 'oat\\taoQtiItem\\model\\qti\\feedback\\Feedback');
+                    $feedback = $collection->getUnique($newIdentifier, static::class);
```
Comment (wazelin) [inline model/qti/feedback/Feedback.php:98]: I don't really know if it makes a difference, but just to be safe, kindly consider replacing the `static` reference with `self`, in order to keep the behavior as it was.
```suggestion
                    $feedback = $collection->getUnique($newIdentifier, self::class);
```

File: model/qti/feedback/ModalFeedback.php (hunk 1 above)
Comment (Alroniks) [reply model/qti/feedback/ModalFeedback.php:54]: I am not sure that it is actually used somewhere, but probably yes. I am fixed only code styles and issues, that IDE highlighted by red across all files. But yes, it makes sense to remove it as unused.

File: model/qti/feedback/Feedback.php (hunk 2 above)
Comment (Alroniks) [reply model/qti/feedback/Feedback.php:98]: Actually it does not matter. But I don't mind.

File: model/qti/feedback/ModalFeedback.php (hunk 1 above)
Comment (Alroniks) [reply model/qti/feedback/ModalFeedback.php]: I removed the method as unused.

File: model/qti/feedback/Feedback.php [hunk 3]
```diff
@@ -59,28 +65,27 @@ public function getBody()
...
         $returnValue = false;
 
         if (empty($newIdentifier) || is_null($newIdentifier)) {
             throw new InvalidArgumentException("newIdentifier must be set");
         }
 
-        if (!empty($this->identifier) && $newIdentifier == $this->identifier) {
+        if (!empty($this->identifier) && $newIdentifier === $this->identifier) {
             $returnValue = true;
         } else {
```
Comment (wazelin) [inline model/qti/feedback/Feedback.php:90]: Let's get rid of this ugly `else` block by introducing an early return, based on the condition above.

**Comment by Alroniks** [comment]
@wazelin I simplified code flow

**Pull request merged**